utxo/src/utxo_impl/simulation.rs

```diff
@@ -112,7 +112,8 @@ fn populate_cache<'a>(
     (cache, outps)
 }
 
-// #[test]
+#[ignore]
+#[test]
```

Comment:
Perhaps I’m missing something, but it seems better to me to mark the test as ignored instead of commenting it.

Reply:
Actually this may be a reminder for looking into this again later because this crate isn't used yet. Marking it ignore shows it's intentional. It's OK to mark it as ignored but please add a TODO to reconsider enabling this test.

Reply:
Unfortunately, the test is failing now, so I have created a separate issue for that: https://github.com/mintlayer/mintlayer-core/issues/219
I can try to fix it, but I will probably need some guidance.